app.rb

```diff
@@ -0,0 +1,40 @@
+require 'sinatra/base'
+require_relative './lib/peeps'
+require './lib/users'
+require_relative './database_connection_setup'
+
+class Chitter < Sinatra::Base
+  enable :sessions
+
+  get '/' do
+    @user = Users.find(session[:user_id])
+    erb :'index'
+  end
+
+  get '/peeps' do
+    @peeps = Peeps.all.sort_by { |a| -a.id.to_i }
```

Comment:
I like that you left your .all method the same and just sorted by using .sort. Well done!